src/python/ensembl/io/genomio/genbank/extract_data.py

```diff
@@ -198,7 +206,35 @@ def _write_genes_gff(self) -> None:
         if num_duplicates > 0:
             raise GBParseError(f"Some {num_duplicates} IDs are duplicated")
 
+    def _write_genes_gff(self, gff_records: List[SeqFeature]) -> None:
```

Comment:
I recommend you change the name from `gff_records` to `gff_features` in this function